.dprint.json

```diff
@@ -0,0 +1,25 @@
+{
+  "incremental": true,
+  "typescript": {
+  },
+  "json": {
+  },
+  "markdown": {
+    "lineWidth": 100
+  },
+  "toml": {
+  },
+  "includes": ["docs/**/*.{ts,tsx,js,jsx,cjs,mjs,json,md,toml}", "**/*.md"],
+  "excludes": [
+    "**/node_modules",
+    "**/*-lock.json",
+    "docs/.vuepress/",
+    ".github/*"
+  ],
+  "plugins": [
+    "https://plugins.dprint.dev/typescript-0.62.0.wasm",
```

Comment:
My thought here was to use the TS/JS/JSON formatters to hit anything in the docs directory that was directly written by us, but it seems that there are no such files (or they all exist in .vuepress).

Do we want to get rid of the typescript and JSON formatters in that case? (The TOML one should stay because it's used to format TOML in Markdown code blocks, which we have a ton of).

Reply:
I don't think speed is an issue at all (maybe if you have the starship directory on a network drive). I just wasn't sure if .vuepress was something we should format. Will add the directory.